components/cluster/pcs/Makefile

```diff
@@ -72,3 +74,4 @@ REQUIRED_PACKAGES += library/ruby/bundler
 REQUIRED_PACKAGES += runtime/python-27
 REQUIRED_PACKAGES += runtime/ruby-23
 REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += system/font/truetype/liberation
```

Comment:
Do I understand correctly, that this font is required during build time?

Reply:
It is used in the webgui.

Reply:
Then likely it should be in package dependencies (depend type=require fmri=...), and not here.